.vscode/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+<form id="form" onsubmit="handleForm('form')">
```

Comment:
@prateeksanodiya0209 If we are passing 'form' to the `handleForm` then you will not able to access the event. Can you please change it to below code?
```
handleForm(event)
```